dependency-check-suppressions.xml

```diff
@@ -106,8 +107,20 @@
         <notes>tomcat-embed-core 9.0.33 suppression, root cause and fix needs to be carried out separately</notes>
         <cve>CVE-2020-17527</cve>
     </suppress>
+
     <suppress>
-        <notes>cpe:2.3:a:google:guava:27.0 suppression, root cause and fix needs to be carried out separately</notes>
+        <notes>Launch-darkley jar uses guava@28.2-jre as an internal library and it is not exposed outside. Since it cannot be upgraded,
+            this supression is added. </notes>
         <cve>CVE-2020-8908</cve>
     </suppress>
+
+    <suppress until="2021-03-01">
```

Comment:
Is this date recommended by RPE?  By specifying a certain date for suppression, how confident we are that some latest Library would be available with fix to the CVEs

Reply:
This specific date is added based on other repositories where this supression has been added

Reply:
This way the build will go fail on 2nd March 2021

Reply:
As discussed with @YasharRahvar, let's have 2 weeks of buffer date for a CVE to be rechecked for it's remediation. So the a suppression on 1st of Jan,2021 would have a date of 15th Jan, 2021